src/pages/index.js

```diff
@@ -9,12 +16,15 @@ export default ({ data }) => {
       <h1>Projects</h1>
       <h4>{projectCount} Posts</h4>
       {data.allMarkdownRemark.edges.map(({ node }) => {
+        let projectSummary = typeof node.frontmatter.summary === 'string' ?
```

Comment:
I see we have projectSummary on index and project, is there a reason it's called projectSummary in both places?

Reply:
Just because for now we need it in both places and it is the same in both places.